package.json

```diff
@@ -100,18 +98,8 @@
     "prettier": "^2.6.1",
     "typescript": "^4.6.3"
   },
-  "config": {
-    "commitizen": {
-      "path": "./node_modules/cz-conventional-changelog"
-    }
-  },
   "lint-staged": {
     "*.{js,ts,tsx,jsx}": "eslint --fix",
     "*.{js,ts,tsx,jsx,md,json}": "prettier --write"
-  },
-  "husky": {
```

Comment:
How come we're removing husky?

Reply:
This was just for using husky with commitizen, husky is still part of the project :)